code-challenges.test.js

```diff
@@ -14,56 +14,177 @@
+    expect(multThree(numbersArray2)).toEqual( [72, 81, 90, 99, 108])
+    })
+})
 
 // b) Create the function that makes the test pass.
+    //Pseudo-Code for Function:
+        // Create a function caled multThree
+        // Take in an array of numbers 
+        // Set up return
+        // Map through the array or loop through the array
+        // Map can take three arguments, first one is value
+        // Map needs a return statement with the value multipled by 3
+        // Return the array of numbers multipled by 3. 
 
+const multThree = (array) => {
+    let mappedArray = array.map(value => value * 3)
+    return mappedArray
+}
 
 
 // --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
 
+
 // a) Create a test with expect statements for each of the variables provided.
 
-const num1 = 15
-// Expected output: "15 is divisible by three"
-const num2 = 0
-// Expected output: "0 is divisible by three"
-const num3 = -7
-// Expected output: "-7 is not divisible by three"
+// const num1 = 15
+// // Expected output: "15 is divisible by three"
+// const num2 = 0
+// // Expected output: "0 is divisible by three"
+// const num3 = -7
+// // Expected output: "-7 is not divisible by three"
+
+//Psuedo-Code for Jest Test:
+    // A describe method that lists the name of the function OR naming of the particular test.
+    // A test/it method, nested within the describe block, that in plain words, describe what the function does.
+    // An expect method, nested within the test block, calling on the divisibleOrNot() function, followed by the toEqual() matcher that checks the expected output of the function return.
+    // Make 3 expects for the 3 const varaibles procided. 
+describe("divisibleOrNot", () => {
+    const num1 = 15
+    const num2 = 0
+    const num3 = -7
+    it("returns whether the number is evenly divisible by three or not", () => {
+        expect(divisibleOrNot(num1)).toEqual("15 is divisble by three")
+        expect(divisibleOrNot(num2)).toEqual("0 is divisble by three")
+        expect(divisibleOrNot(num3)).toEqual("-7 is not divisble by three")
+    })
+})
 
 
 // b) Create the function that makes the test pass.
+    //Pseudo-Code for Function:
+        // Function Signature:
+            //Input: a number
+            // Output: Returns a string stating Whether the number is evenly divisble by 3 or not. 
+        //Process:
+            // Create a function called divisibleOrNot taking in a number as an argument, call the parameter number1.
+            // Use a conditional statement (if/else) and return whether the number is divisible by 3 or not. 
+            // Will have to use modules in order to find the divisibility.
+const divisibleOrNot = (number1) => {
+    if(number1 % 3 === 0) {
+        return `${number1} is divisble by three`
+    } else {
+        return `${number1} is not divisble by three`
+    }
+}
```

Comment:
Excellent!  If you want to stretch your knowledge on if-else statements, take a look at ternary operators.